website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -0,0 +1,574 @@
+
+**Note**: For those familiar with the SDK's old project semantics, [Handle][doc-osdk-handle] received resource events and reconciled state for multiple resource types, whereas Reconcile receives resource events and reconciles state for a single resource type.
+
+```Go
+// ReconcileKind reconciles a Kind object
+type ReconcileKind struct {
+	// client, initialized using mgr.Client() above, is a split client
+	// that reads objects from the cache and writes to the apiserver
+	client.Client
+
+    Log    logr.Logger
+
+	// scheme defines methods for serializing and deserializing API objects,
+	// a type registry for converting group, version, and kind information
+	// to and from Go schemas, and mappings between Go schemas of different
+	// versions. A scheme is the foundation for a versioned API and versioned
+	// configuration over time.
+    Scheme *runtime.Scheme
+}
+
+// Reconcile watches for Events and reconciles cluster state with desired
+// state defined in the method body.
+// The Controller will requeue the Request to be processed again if an error
+// is non-nil or Result.Requeue is true, otherwise upon completion it will
+// remove the work from the queue.
+func (r *ReconcileKind) Reconcile(request reconcile.Request) (reconcile.Result, error)
+```
+
+Reconcile is where Controller business logic lives, i.e. where Client API calls are made via `ReconcileKind.client`. A `client.Client` implementer performs the following operations:
+
+#### Get
+
+```Go
+// Get retrieves an API object for a given object key from the Kubernetes cluster
+// and stores it in obj.
+func (c Client) Get(ctx context.Context, key ObjectKey, obj runtime.Object) error
+```
+**Note**: An `ObjectKey` is simply a `client` package alias for [`types.NamespacedName`][doc-types-nsname].
+
+Example:
+```Go
+import (
+	"context"
+	"github.com/example-org/app-operator/pkg/apis/cache/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	...
+
+	app := &v1alpha1.App{}
+	ctx := context.TODO()
+	err := r.Get(ctx, request.NamespacedName, app)
+
+	...
+}
+```
+
+#### List
+
+```Go
+// List retrieves a list of objects for a given namespace and list options
+// and stores the list in obj.
+func (c Client) List(ctx context.Context, list runtime.Object, opts ...client.ListOption) error
+```
+
+A `client.ListOption` is an interface that sets [`client.ListOptions`][list-options] fields. A `client.ListOption` is created by using one of the provided implementations: [`MatchingLabels`][matching-labels], [`MatchingFields`][matching-fields], [`InNamespace`][in-namespace].
+
+Example:
+
+```Go
+import (
+	"context"
+	"fmt"
+	"k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Update for new scaffold. `request reconcile.Request`==>`req ctrl.Request`
Update the imports as well `"sigs.k8s.io/controller-runtime/pkg/reconcile"`==>`ctrl "sigs.k8s.io/controller-runtime"`